base-server.d.ts

```diff
@@ -0,0 +1,704 @@
+import { Log, Store, ServerConnection, TestTime } from '@logux/core'
+import { Server as HTTPServer } from 'http'
+
+import { Context, ChannelContext, LoguxPatternParams } from './context'
+import { ServerClient } from './server-client'
+
+/**
+ * Logux meta
+ */
+export type LoguxMeta = {
```

Comment:
Logux Core already has `Meta`. Can we extend it as we do in Logux Client?
https://github.com/logux/client/blob/master/client/index.d.ts#L11
https://github.com/logux/client/blob/master/client/index.d.ts#L168

Also, note that `meta` can contain any other extra fields.